src/extras/core/CurvePath.js

```diff
@@ -66,7 +66,13 @@ THREE.CurvePath.prototype = Object.assign( Object.create( THREE.Curve.prototype
 				var diff = curveLengths[ i ] - d;
 				var curve = this.curves[ i ];
 
-				var u = 1 - diff / curve.getLength();
+				var segmentLength = curve.getLength();
+				var u = 0;
+				if ( segmentLength !== 0 ) {
+
+				  u = 1 - diff / curve.getLength();
```

Comment:
I added this bloc because curve.getLength() can be 0. For example, put both angle on the same value in my [fiddle,](https://jsfiddle.net/wdsmy9k3/3/) getLength return 0 and the red point goes to the origin because a division by 0 occurs.
I’m calling getLength twice because I did finish my idea of using the segmentLength I defined not to use getLength twice. It’s a bug, your suggestion is the right line of code.